### PR TITLE
move validators & normalizers to tensormap/

### LINKED ISSUE #1218: Move validators and normalizers into /tensormap
## What and why
https://github.com/aguirre-lab/ml4c3/discussions/1176

## Solution(s)
Organize scripts.

## Acceptance criteria
`validators.py` and `normalizers.py` are in `/tensormap` and import statements are updated throughout codebase.

### PR DESCRIPTION
**Pull request by erikr**
## Issues closed
Closes #1218

## Changelog
- [x] `validators.py` and `normalizers.py` are in `/tensormap` and import statements are updated throughout codebase.
- [x] Regenerate `tensormap/ecg-labels.py`.

## Checklist
- [x] The following are complete:
    - Added or revised tests if appropriate.
    - If new packages are installed, `requirements.txt` is updated.
    - If documentation should be updated, a new issue was made in `ml4c3-wiki`.
    - The PR name describes its content and does not contain "Ref #".
    - The PR does not add PHI.

### CODE REVIEW

File: docker/config/requirements.txt [hunk 1]
```diff
@@ -11,20 +12,19 @@ ipywidgets>=7.5.1
...
+scikit-learn
 seaborn>=0.10.1
 tabulate
-tensorflow-probability==0.11.1
 tensorflow-addons==0.11.2
+tensorflow-probability==0.11.1
 tqdm
 unidecode
 xgboost
-xlrd
```
Comment (raimonpv) [inline docker/config/requirements.txt:28]: we are not using this package anymore?

File: tensormap/ecg_labels.py [hunk 2]
```diff
@@ -6,16 +6,16 @@
...
+from ml4c3.definitions import ECG_PREFIX
+from tensormap.validators import validator_not_all_zero
 
 tmaps: Dict[str, TensorMap] = {}
 tmaps["asystole"] = TensorMap(
     "asystole",
     interpretation=Interpretation.CATEGORICAL,
     time_series_limit=0,
-    path_prefix=ECG_PREFIX,
+    path_prefix="ECG_PREFIX",
```
Comment (raimonpv) [inline tensormap/ecg_labels.py:18]: is that on purpose?

**Review by raimonpv — APPROVED** [review]

File: tensormap/ecg_labels.py (hunk 2 above)
Comment (erikr) [reply tensormap/ecg_labels.py:18]: Good catch, this is a mistake.

File: docker/config/requirements.txt (hunk 1 above)
Comment (erikr) [reply docker/config/requirements.txt:28]: The default engine `xlrd` used by `pd.read_excel` no longer supports `.xlsx` files, which is the format of our label maps used to generate `ecg-label.py` (https://github.com/aguirre-lab/ml4c3/wiki/TensorMaps#build-tmaps-for-labels-from-ecg-reads).

Instead, the new default engine is `openpyxl`, but this must be installed.

See https://github.com/pandas-dev/pandas/issues/38424 for more info about this change.

**Pull request merged**